### DISCUSSION TITLE
Listbox.Option render prop "active" doesn't work when Listbox.Options has static={true}

### DISCUSSION THREAD

**Discussion by tfilo** [tailwindlabs/headlessui#2487] [Help]
Hello, I work on customized Listbox where options are always expanded. I used static prop on <Listbox.Options> tag to achieve that. But when I have static={true}, than render prop active on individual Listbox.Option is always false. Can you help me how to solve this ? Thanks.

Simplified code example:

```JSX
<Listbox
   value={[]}
   onChange={() => void}
   multiple={true}
   as='div'
>
   <Listbox.Options static={true}>
      {options.map((o) => (
         <Listbox.Option
            key={o.id}
            className={({ active }) =>
               `${
                  active ? 'some classes when active' : 'some classes when not active'
               }`
            }
            value={o}
         >
            {o.name}
         </Listbox.Option>
      ))}
   </Listbox.Options>
</Listbox>
```

**Comment by lancechentw** [comment]
I'm trying to do the same thing. I think `static` is not the reason why `active` doesn't work, it's probably because `activeOptionIndex` is only set when certain events occur (most are bound to `Listbox.Button`). 

https://github.com/tailwindlabs/headlessui/blob/1469b85c36802265c2409f443f926e1bb02230d4/packages/%40headlessui-react/src/components/listbox/listbox.tsx#L153-L158

My current hack is to leave `Listbox.Button` there, and have it autoFocus on component loaded, so when a user press arrow up, arrow down, space, or enter, the button's event set a value to `activeOptionIndex`.
```diff
 <Listbox
    value={[]}
    onChange={() => void}
    multiple={true}
    as='div'
 >
+   <Listbox.Button autoFocus></Listbox.Button>
    <Listbox.Options static={true}>
       {options.map((o) => (
          <Listbox.Option
             key={o.id}
             className={({ active }) =>
                `${
                   active ? 'some classes when active' : 'some classes when not active'
                }`
             }
             value={o}
          >
             {o.name}
          </Listbox.Option>
       ))}
    </Listbox.Options>
 </Listbox>
```
